packages/teleterm/src/ui/TopBar/Connections/ConnectionsFilterableList/ConnectionItem.tsx

```diff
@@ -48,37 +48,43 @@ export function ConnectionItem(props: ConnectionItemProps) {
         height: unset;
       `}
     >
-      <ConnectionStatusIndicator mr={3} connected={props.item.connected} />
+      <ConnectionStatusIndicator
+        mr={3}
+        css={`
+          flex-shrink: 0;
+        `}
+        connected={props.item.connected}
+      />
       <Flex
         alignItems="center"
         justifyContent="space-between"
         flex="1"
-        width="100%"
```

Comment:
`flex: 1` and `width: 100%` seem unecessary. Instead of taking 100% width of its parent and letting other elements cut into it (like `ConnectionStatusIndicator` did here), let's use flexbox to its fullest: make `ConnectionStatusIndicator` never shrink and make this div here take all remaining space with `flex: 1`.